src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.AssemblyResolution.targets

```diff
@@ -49,7 +52,6 @@ _ResolveAssemblies MSBuild target.
       <ItemGroup>
         <ResolvedFileToPublish Remove="@(_SourceItemsToCopyToPublishDirectory)" />
         <ResolvedFileToPublish
-            Condition=" '%(ResolvedFileToPublish.RuntimeIdentifier)' == '' "
             Update="@(ResolvedFileToPublish)"
             RuntimeIdentifier="$(RuntimeIdentifier)"
         />
```

Comment:
If you are using at least a .NET 10 Preview 2 runtime, you can remove this change.

It should successfully use Android RIDs now, and we don't need to replace the `linux-bionic` RID that was here previously.